PatchKit/Classes/Controllers/CustomerServiceDetailViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  CustomerServiceDetailViewController.swift
+//  PatchKit
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import UIKit
+
+class CustomerServiceDetailViewController: UIViewController {
+    
+    private let addImageButton = UIButton()
+    private let bottomBar = UIView()
+    private let messagesTableView = UITableView()
+    private let messageTextView = UITextView()
+    private let sendButton = UIButton()
+    
+    private var messageThread: MessageThread!
+    private var messageTextViewHeightConstraint: NSLayoutConstraint!
```

Comment:
we should make these optional

Reply:
Should the VC be able to exist without them though?

Reply:
well ideally its always there but making it optional is safer. if somehow its not there, it wont crash